en/docs/develop/customizations/customizing-the-developer-portal/styling-api-details-info-section.md

```diff
@@ -1,16 +1,20 @@
 # Styling API Details Info Section
 
-The API details info section can be customized to match with your design needs.
+The "API Details Info" section shown below, can be customized according to your design needs.
 
- ![styling api details info section]({{base_path}}/assets/img/learn/styling-api-details-info-section1.png) 
+ ![styling api details info section]({{base_path}}/assets/img/learn/styling-api-details-info-section1.png)
 
-You can change themes.light.custom.infoBar attributes to change the api detials info section styling. Note these changes will effect the same way to application details info section.
+Edit the attributes in `themes.light.custom.infoBar` to change the styling "API Details Info" section. Note these changes will effect the same way to application details info section.
```

Comment:
Change [1] to [2]

[1] "Edit the attributes in `themes.light.custom.infoBar` to change the styling "API Details Info" section."
[2] Edit the attributes in `themes.light.custom.infoBar` to change the styling "API Details Info" and the "Application Details Info" section.